npm/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "undefined",
```

Comment:
Just cosmetic I think, but name this ^ "runway" I think

Reply:
fyi, I set it to undefined to default away from runway. The name is set based on a travis env var when preparing to publish allowing forks to publish a clone from their fork as needed. but, we can default it to runway.

https://github.com/ITProKyle/runway/blob/feature/single-binary/Makefile#L57

Reply:
defaulted name to `runway` in package.json